File: simpleinheri.java
class Vehicle {
    public void engine() {
        System.out.println("the vehicle's engine is a petrol engine");
    }
    int vehicleNum = 1234;
}

class Car extends Vehicle {
    String color = "red";
}

public class inherit {
    public static void main(String[] args) {
        Car obj = new Car();
        obj.engine();
        System.out.println(obj.vehicleNum);
        System.out.println(obj.color);
    }
}

File: simpleinheri.cpp
#include<iostream>
using namespace std;
class vehicle{
    public:
    void enginee(){
        cout<<"the vehicle's enginee is a petrol enginee"<<endl;
    }
    int vehiclenum=1234;
};
class car:public vehicle{
    public:
    string color="red";
};
int main(){
    car obj;
    obj.enginee();
    cout<<obj.vehiclenum<<endl;
    cout<<obj.color<<endl;
}

